juriscraper/pacer/attachment_page.py

```diff
@@ -90,23 +90,28 @@ def data(self):
             result["file_size_str"] = ""
         else:
             result["page_count"] = self._get_page_count_from_tr(first_row)
+            file_size_bytes = self._get_file_size_bytes_from_tr(first_row)
+            if file_size_bytes is not None:
+                result["file_size_bytes"] = file_size_bytes
             result["file_size_str"] = self._get_file_size_str_from_tr(
                 first_row
             )
         for row in rows:
-            result["attachments"].append(
-                {
-                    "attachment_number": self._get_attachment_number(row),
-                    "description": self._get_description_from_tr(row),
-                    "page_count": self._get_page_count_from_tr(row),
-                    "file_size_str": self._get_file_size_str_from_tr(row),
-                    "pacer_doc_id": self._get_pacer_doc_id(row),
-                    # It may not be needed to reparse the seq_no
-                    # for each row, but we may as well. So far, it
-                    # has always been the same as the main document.
-                    "pacer_seq_no": self._get_pacer_seq_no_from_tr(row),
-                }
-            )
+            attachment = {
+                "attachment_number": self._get_attachment_number(row),
+                "description": self._get_description_from_tr(row),
+                "page_count": self._get_page_count_from_tr(row),
+                "file_size_str": self._get_file_size_str_from_tr(row),
+                "pacer_doc_id": self._get_pacer_doc_id(row),
+                # It may not be needed to reparse the seq_no
+                # for each row, but we may as well. So far, it
+                # has always been the same as the main document.
+                "pacer_seq_no": self._get_pacer_seq_no_from_tr(row),
+            }
+            file_size_bytes = self._get_file_size_bytes_from_tr(row)
+            if file_size_bytes is not None:
+                attachment["file_size_bytes"] = file_size_bytes
```

Comment:
It's usually better to have the key in the dictionary, even if it's `None`. That way downstream can just do:

```
filesize = att["filesize_bytes"]
```

Instead of:
```
filesize = att.get("filesize_bytes")
```

I guess it's a small thing, but that's usually how we do it.

Reply:
Well we still have to fall back to parsing `file_size_str` when `file_size_bytes` isn't present...so...I don't think that's all that helpful for this particular case.

Reply:
Very well. At some point we should do a json schema for this stuff.